teammapper-frontend/mmp/src/map/handlers/history.ts

```diff
@@ -26,6 +27,28 @@ export default class History {
         this.snapshots = []
     }
 
+    /**
+     * As snapshots are saved by number (1, 2, 3 etc) and not by node ID, this switches out the number with the affected node ID.
```

Comment:
thank you, this already helps a lot. If you want to go through a list and exchange keys (number, 1,2,3... with the content) with other keys (nodeId), that sounds like a good task for https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

I think the code would be easier readable with this. What do you think?

Reply:
`map()` probably won't work since we're changing the array in place and not creating a new one, but I've tried to simplify the method a bit to hopefully make it a bit more readable.